fvm/transactionSequenceNum.go

```diff
@@ -21,15 +23,25 @@ func (c *TransactionSequenceNumberChecker) Process(
 	programs *Programs,
 ) error {
 
-	return c.checkAndIncrementSequenceNumber(proc.Transaction, st)
+	return c.checkAndIncrementSequenceNumber(proc, ctx, st)
 }
 
 func (c *TransactionSequenceNumberChecker) checkAndIncrementSequenceNumber(
-	tx *flow.TransactionBody,
+	proc *TransactionProcedure,
+	ctx Context,
 	st *state.State,
 ) error {
+
+	if ctx.Tracer != nil && proc.TraceSpan != nil {
+		span := ctx.Tracer.StartSpanFromParent(proc.TraceSpan, trace.FVMSeqNumCheckTransaction)
+		span.LogFields(
+			log.String("transaction.hash", proc.ID.String()),
```

Comment:
Maybe `transaction.ID` to keep things consistent?

Reply:
applied.